src/app-layout/notifications/styles.scss

```diff
@@ -4,9 +4,11 @@
 */
 
 @use '../../internal/styles/tokens' as awsui;
+@use '../../internal/generated/custom-css-properties/index.scss' as custom-props;
 
 .notifications-sticky {
   top: 0;
   z-index: 825;
   position: sticky;
+  #{custom-props.$flashbarStickyBottomMargin}: #{awsui.$space-xxl};
```

Comment:
How does this work? 
Does the left side get replaced with the name for a CSS custom property/variable (`--something-like-this`)?

Reply:
Yes. Having a custom property directly written here is not allowed by our linting, so I added a new one `flashbarStickyBottomMargin` in `internal/generated/custom-css-properties/list.js` (this file itself doesn't seem to be generated, but it is used for the generation of custom properties)